deep-learning/_posts/2018-06-01-MDRNN.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title:  "Multimodal Recurrent Neural Networks with Information Transfer Layers for Indoor Scene Labeling"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1803.04687.pdf
+cite:
+authors: "Abrar H. Abdulnabi, Bing Shuai, Zhen Zuo, Lap-Pui Chau, Gang Wang"
+title:   " Indoor Scene Labeling"
```

Comment:
Please put the complete title of the paper as well as the complete arXiv reference.

Reply:
I made mistake about the name after I push it, I found.